assembly/src/tests.rs

```diff
@@ -2198,13 +2198,7 @@ fn invalid_proc_duplicate_procedure_name() {
         source,
         "syntax error",
         "help: see emitted diagnostics for details",
-        "symbol conflict: found duplicate definitions of the same name",
-        regex!(r#",-\[test[\d]+:1:6\]"#),
-        "1 | proc.foo add mul end proc.foo push.3 end begin push.1 end",
-        "  :      ^|^             ^^^^^^^^^|^^^^^^^^^",
-        "  :       |                       `-- conflict occurs here",
-        "  :       `-- previously defined here",
-        "  `----"
+        "procedure name conflict: found duplicate definitions of 'foo'"
```

Comment:
Why do we lose this nice error here?

Reply:
Restored previous behavior; this was related to https://github.com/0xPolygonMiden/miden-vm/pull/1401#discussion_r1688523832